PREPARE_RELEASE.md

```diff
@@ -0,0 +1,48 @@
+# Prepare for a new release
+
+In this repo you will need to perfrom the following tasks manually
+
+There is a lot of information on what is needed to manage OLM [compatible operators](https://redhat-connect.gitbook.io/certified-operator-guide/ocp-deployment/operator-metadata/creating-the-csv).
+
+- Update `version/version.go` bumping the minor or patch version. Major versions shouldn't be released without discussion and approval of the maintainers!
+- Update the `Makefile`
+- Update `replaces` field in `config/manifests/bases/grafana-operator.clusterserviceversion.yaml`
+- Update `CreatedAt` field in `config/manifests/bases/grafana-operator.clusterserviceversion.yaml`
+  You will have to asses when it's going to get merged and you will be able to do a release.
+  You should make sure it's the same date. If not you will have to change it
+  manually when creating PR:s to OLM.
+
+      # This is how the time syntax should look.
+      $ docker inspect quay.io/grafana-operator/grafana-operator:v4.0.2 |jq '.[0].Created'
+      "2021-11-22T10:34:12.173861869Z"
+      # 2021-11-22T10:34:12Z is enough
+- Run `make bundle`
+- Run `make bundle-kustomization KUSTOMIZE_TAG=v4.0.2` the KUSTOMIZE_TAG should match the new tag
```

Comment:
Interesting, i never used this. The image was updated after only running `make bundle`

Reply:
I know, It's brand new :) 
It will start to exist and will be needed when: https://github.com/grafana-operator/grafana-operator/pull/624 get's merged.